### PR TITLE
chore: update creation deposit from silo

### PR DESCRIPTION
**Pull request by jobotics**

### CODE REVIEW

File: src/constants/tokens.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import type { BaseTokenInfo } from "../types/base"
 
+export const NATIVE_TOKEN_ADDRESS = "0x0000000000000000000000000000000000000000"
```
Comment (cawabunga-bytes) [inline src/constants/tokens.ts:3]: It's exclusively for AuroraEngine SiloToSilo contract, I suggest putting in place where it's used.

File: src/features/deposit/components/DepositUIMachineProvider.tsx [hunk 2]
```diff
@@ -240,12 +240,14 @@ export function DepositUIMachineProvider({
                 )
 
                 const tx = createDepositFromSiloTransaction(
-                  tokenAddress,
+                  tokenAddress === "native"
```
Comment (cawabunga-bytes) [inline src/features/deposit/components/DepositUIMachineProvider.tsx:243]: `tokenAddress` won't be `native` in the future.

**Review by cawabunga-bytes — APPROVED** [review]
`tokenAddress`

**Comment by cawabunga-bytes** [comment]
Fix types please

File: src/features/deposit/components/DepositUIMachineProvider.tsx (hunk 2 above)
Comment (jobotics) [reply src/features/deposit/components/DepositUIMachineProvider.tsx:243]: noted

**Pull request merged**